CedarJava/build.gradle

```diff
@@ -187,10 +178,20 @@ tasks.register('uberJar', Jar) {
 }
 
 tasks.named('test') {
+    useJUnitPlatform()
     dependsOn('compileFFI')
     classpath += files(layout.buildDirectory.dir(compiledLibDir))
 }
 
+test {
+    //environment "CEDAR_INTEGRATION_TESTS_ROOT", ''set to absolute path of `cedar-integration-tests`'
```

Comment:
Ok to delete this line? We already mention that the env variable needs to be set in the readme

Reply:
At least personally, I prefer to set this here and then run builds in Intellij rather than exporting this variable in a terminal.